app/domain/operations/notices/ivl_enr_notice_trigger.rb

```diff
@@ -121,12 +123,11 @@ def build_enrollments_hash(enrollments)
             hbx_enrollment_members: enrollment_member_hash(enr),
             product_reference: product_reference(product, issuer),
             issuer_profile_reference: issuer_profile_reference(issuer),
-            consumer_role_reference: consumer_role_reference(consumer_role),
             is_receiving_assistance: (enr.applied_aptc_amount > 0 || (product.is_csr? ? true : false)),
             timestamp: timestamp(enr)
           }
           enrollment_hash.merge!(special_enrollment_period_reference: special_enrollment_period_reference(enr)) if enr.is_special_enrollment?
-          enrollment_hash
+          enrollment_hash.merge!(consumer_role_ref || resident_role_ref)
```

Comment:
We should add all roles that present on the enrollment, i.e., add both consumer role and resident role if both exist on the enrollment.